### ISSUE TITLE
Adapt MobDebug debugger

### ISSUE THREAD

**Issue by rtsisyk** [tarantool/tarantool#2728]
A [MobDebug](https://github.com/pkulchenko/MobDebug) is remote debugger for Lua and LuaJIT, used by ZeroBrain Studio and Lua plugins for IntelliJ. Please add required dependencies needed for MobDebug to Tarantool core OR patch debugger itself.

**Comment by rtsisyk** [comment]
Blocked by #2727

**Comment by rtsisyk** [comment]
Already done.

**Comment by Fedora7** [comment]
If I put  require('mobdebug').start("ip address") in a abc.lua file or lua module,  
and then run it by dofile('abc.lua') or require('abc') on tarantool server's console ,  
the debugger doesn't stop for debugging.

But if I run it as one time script in command line:  tarantool abc.lua , it worked and connected to zeroBrane studio for debugging.

Can it be the way how MobDebug determine "main" in lua ? or it's not compatible with tarantool fiber since the MobDebug use luajit native coroutine inside ? Thanks
